src/test/java/com/puppycrawl/tools/checkstyle/AbstractAutomaticBeanTest.java

```diff
@@ -278,6 +280,46 @@ public void testBeanConvertersUri3() {
         }
     }
 
+    @Test
+    public void testBeanConverterPatternArray1() throws Exception {
+        final ConverterBean bean = new ConverterBean();
+        final DefaultConfiguration config = new DefaultConfiguration("bean");
+        final String patternString = "^a*$,^b*$, ^c*$";
+        final List<String> expectedPatternString = Arrays.asList("^a*$", "^b*$", " ^c*$");
```

Comment:
`expectedPatterns` and make conversion to Petterns here as you did in assert

Reply:
done